docs/pages/_includes/services/security-transport-encryption-clients.md

```diff
@@ -0,0 +1,3 @@
+#### Transport Encryption Clients
+
+With Transport Encryption enabled, service clients will need to be configured to use [the DC/OS CA bundle](https://docs.mesosphere.com/latest/security/ent/tls-ssl/get-cert/) to verify the connections they make to the service. Consult your client's documentation for trusting a CA and configure your client appropriately.
```

Comment:
Is it "**T**ransport **E**ncryption" or just "**t**ransport **e**ncryption"? I prefer the latter.

Reply:
I've been calling it Transport Encryption everywhere to make it more of a "thing"